Inc/GraphicsMemory.h

```diff
@@ -78,7 +78,7 @@ namespace DirectX
             LinearAllocatorPage*        mPage;
             D3D12_GPU_VIRTUAL_ADDRESS   mGpuAddress;
             ID3D12Resource*             mResource;
-            void*                       mMemory;
+            ID3D12Resource*             mMemory;
```

Comment:
The `mMemory` and the `mResource` are not the same thing. The `mResource` is a control object in the runtime, not the memory being managed.